packages/slice-machine/components/Simulator/components/Header/index.tsx

```diff
@@ -102,56 +69,35 @@ const Header: React.FunctionComponent<PropTypes> = ({
     <Box
       sx={{
         p: 3,
-        display: "grid",
+        display: "flex",
         gridTemplateColumns: "repeat(3, 1fr)",
         gridTemplateRows: "1fr",
         borderBottom: "1px solid #F1F1F1",
+        justifyContent: "space-between",
       }}
     >
       <Flex
         sx={{
           alignItems: "center",
         }}
       >
-        <Text mr={2}>{Model.model.name}</Text>
-        {Model.model.variations.length > 1 ? (
-          <VarationsPopover
-            buttonSx={{ p: 1 }}
-            defaultValue={variation}
-            variations={Model.model.variations}
-            onChange={(v) => redirect(Model, v, true)}
-          />
-        ) : null}
+        <SliceMachineLogo height={"20px"} width={"20px"} />
+        <Text mx={2}>{Model.model.name}</Text>
       </Flex>
-      <Flex
-        sx={{
-          alignItems: "center",
-          justifyContent: "space-around",
-        }}
-      >
-        <ScreenSizes size={size} onClick={handleScreenSizeChange} />
-      </Flex>
-      <Flex
-        sx={{
-          alignItems: "flex-end",
-          flexDirection: "column",
-        }}
-      >
-        <Button
-          onClick={onTakingSliceScreenshot}
-          label="Take a screenshot"
-          isLoading={isSavingScreenshot}
-          Icon={AiFillCamera}
+      <Button
+        onClick={onTakingSliceScreenshot}
+        label="Take a screenshot"
+        isLoading={isSavingScreenshot}
+        Icon={AiFillCamera}
+      />
+      {isWaitingForIframeCheck && (
+        <IframeRenderer
+          dryRun
```

Comment:
❓ I was wondering, do we need this ?
We already have the iframeRenderer on the Simulator/Index.tsx

Reply:
You're right I think. It's probably redundant here